tests/integration/remote/test_mumbai.py

```diff
@@ -21,41 +18,19 @@ def test_nonocean_tx(tmp_path):
     ocean = Ocean(config)
     (alice_wallet, bob_wallet) = get_wallets(ocean)
 
-    # Get gas price (in Gwei) from Polygon gas station
-    gas_station_url = "https://gasstation-mumbai.matic.today/v2"
-    gas_price = requests.get(gas_station_url).json()["fast"]["maxFee"]
-
     # Simplest possible tx: Alice send Bob some fake MATIC
     web3 = ocean.web3
     bob_eth_before = web3.eth.get_balance(bob_wallet.address)
-    nonce = web3.eth.getTransactionCount(alice_wallet.address)
-
-    # avoid "replacement transaction underpriced" error: make each tx diff't
-    normalized_unixtime = time.time() / 1e9
-    amt_send = 1e-8 * (random.random() + normalized_unixtime)
```

Comment:
We need to make it such that each time the test is run, it's a unique tx. Otherwise the EVM will think you're trying to send the same tx, and you will have issues with nonces, and we will see issues #998 and #1007(b) again.

The easiest way to guarantee a unique tx is to send a unique amount each time. At first I just used random.random(), but that was not enough because it may have the same random seed. So I also added the unixtime. That's how I solved those issues above.

How to fix it here: First, re-insert:
```python
    normalized_unixtime = time.time() / 1e9
    amt_send = 1e-8 * (random.random() + normalized_unixtime) #in ether
```

Then use `amt_send` in `send_ether()`. E.g.:
```python
  send_ether(config, alice_wallet, bob_wallet.address, f"{amt_send:.15f} ether")
```